### PR TITLE
Readme: show how to `npm run dev` on another port.

### PR DESCRIPTION
**Pull request by idupree**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default () => (
 )
 ```
 
-and then just run `npm run dev` and go to `http://localhost:3000`
+and then just run `npm run dev` and go to `http://localhost:3000`.  For another port, run `npm run dev -- -p 3001`.
```
Comment (timneutkens) [inline readme.md]: > To use another port, you can run `npm run dev -- -p <your port here>`.

Comment (timneutkens) [inline readme.md]: This might be better

Comment (idupree) [inline readme.md]: Sure. Sent as #1544

Comment (timneutkens) [inline readme.md]: You could just push to the same branch 😄 No worries I'll close this one 👍

**Pull request closed without merging**